roles/kubernetes-apps/csi_driver/upcloud/templates/upcloud-csi-controller.yml.j2

```
@@ -58,13 +58,26 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: csi-snapshotter
+          image: k8s.gcr.io/sig-storage/csi-snapshotter:v4.2.1
```

```suggestion
          image: k8s.gcr.io/sig-storage/csi-snapshotter:{{ upcloud_csi_snapshotter_image_tag }}
```
add variable `upcloud_csi_snapshotter_image_tag` at `roles/kubernetes-apps/csi_driver/upcloud/defaults/main.yml`

---

Done :+1: